app/api/events.py

```diff
@@ -46,7 +46,7 @@
 from app.models.ticket_holder import TicketHolder
 from app.models.track import Track
 from app.models.user_favourite_event import UserFavouriteEvent
-from app.models.user import User, ATTENDEE, ORGANIZER, COORGANIZER
+from app.models.user import User, ATTENDEE, OWNER, ORGANIZER, COORGANIZER
```

Comment:
@shreyanshdwivedi The basics of event an seems to be working fine. Although you might want to create a follow up PR on the front end regarding this (for existing and new events):
![image](https://user-images.githubusercontent.com/21087061/60417177-5fee6180-9bfd-11e9-97d2-2a8fc46e2bbe.png)

Reply:
@uds5501 Yeah, a follow-up PR is required to change few more things on frontend. I'll do it once this PR is merged

Reply:
@shreyanshdwivedi Apologies for not elaborating.
There should be a role of "owner" displayed on the dashboard and the manage event table

Reply:
Sure @uds5501. Will open a followup PR on FE once this is merged. Thanks for the input